src/walletSdk/Horizon/transaction/TransactionBuilder.ts

```diff
@@ -0,0 +1,67 @@
+import StellarSdk, {
+  TransactionBuilder as StellarTransactionBuilder,
+  Account as StellarAccount,
+  Transaction,
+  Server,
+  Memo,
+  xdr,
+  Networks,
+} from "stellar-sdk";
+
+import { Config } from "walletSdk";
+import { AccountKeypair } from "../Account";
+import { InsufficientStartingBalanceError } from "../../Exceptions";
+
+export class TransactionBuilder {
+  private network: Networks;
+  private operations: Array<xdr.Operation>;
+  private builder: StellarTransactionBuilder;
+
+  sourceAccount: string;
```

Comment:
this may live in CommonTransactionBuilder like it does in the [kotlin sdk](https://github.com/stellar/kotlin-wallet-sdk/blob/cf6ab208aa9ab2b60e0a691005ced6ba6f4219d7/wallet-sdk/src/main/kotlin/org/stellar/walletsdk/horizon/transaction/CommonTransactionBuilder.kt#L12) after adding sponsoring code. Still need to think through how it will work in this sdk though